### ISSUE TITLE
Training Details (Epoch)

### ISSUE THREAD

**Issue by dsshim0125** [sshan-zhao/GASDA#14]
Hi,

I'm trying to train the entire network from the scratch.

However, there is no information about the training epoch at each procedure like pretraining F_s, F_t model and fine-tuning GASDA.

The paper says that it takes 20 epochs to pre-train F_s, F_t networks and 40 epochs to finetune the entire network in end to end manner.

The repository says that the training method is different from the paper and also your providing pre-trained model also have a different performance from the paper.

Can you let me know the training epoch of pre-training F_s, F_t and finetuing GASDA?

Thank you.

**Comment by sshan-zhao** [comment]
The default training epoch is set to 110 in the code. Actually, the training epoch is identical to the paper (20/40, I forgot to re-set the parameter). In addition, you can also select the final model according to the performance on the validation set.

**Comment by dsshim0125** [comment]
Thanks a lot!
